### PR TITLE
website/docs: Enterprise docs

### PR DESCRIPTION
**Pull request by tanberry**
Draft of Enterprise docs, still working on them... Ignore sections below TO BE WRITTEN. :-)

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [ ] The documentation has been updated
-   [x] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/docs/enterprise/entsupport.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Enterprise authentik provides dedicated support, with a Support center where you can open a request and view the progress and communications for your current requests.
+
+### Managing tickets and requests
+
+To access the Requests page, where you can open a request and view current requests, go to the Customer Portal and then click **Support** in the top menu.
+
+You can also bookmark the direct link to your Requests page, using the following URL:
+
+> <https://authentiksecurityinc.zendesk.com/hc/en-us/requests>.
```
Comment (BeryJu) [inline website/docs/enterprise/entsupport.md:13]:
```suggestion
> <https://customers.goauthentik.io/l/support>.
```

This allows us to easier change the link in the future

File: website/docs/enterprise/get-started.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+title: Get started
+---
+
+Installing authentik is exactly the same process for both Enterprise version and our free open source version.
+
+> This preview version of Enterprise authentik is available within our 2023.8.x release and later.
+
+## Install Enterprise
+
+To get started working with the preview edition of Enterprise authentik, upgrade to the [2023.8.x version](../releases). For installation steps, [technical documentation](../installation/index.md) for instructions to install and configure authentik.
```
Comment (rissson) [inline website/docs/enterprise/get-started.md:11]:
```suggestion
To get started working with the preview edition of Enterprise authentik, upgrade to the [2023.8.x version](../releases) or later. For installation steps, refer to the [technical documentation](../installation/index.md) for instructions to install and configure authentik.
```

File: website/docs/enterprise/manage-enterprise.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+:::info
+If you need to delete an organization open a ticket in the Support center.
+:::
+
+### Add/remove members of an organization
+
+In the customer portal you can remove members and invite new members to the organization. When you invite new members, you can specify the role for the new member.
+
+-   **Membe**r: can view licenses, including the license key.
```
Comment (rissson) [inline website/docs/enterprise/manage-enterprise.md:27]:
```suggestion
-   **Member**: can view licenses, including the license key.
```

File: website/sidebars.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ const docsSidebar = {
         {
             type: "category",
             label: "Installation",
-            collapsed: false,
+            collapsed: true,
```
Comment (rissson) [inline website/sidebars.js:16]: I liked this one uncollapsed, it avoids searching where the installation and config page are

**Review by rissson — CHANGES_REQUESTED** [review]

File: website/docs/enterprise/licenses-page-admin.png
Comment (BeryJu) [inline website/docs/enterprise/licenses-page-admin.png:1]: Could you use a screenshot that's a little wider, with this resolution the "Go to customer portal" button is slightly cut off

File: website/docs/enterprise/entsupport.md (hunk 1 above)
Comment (tanberry) [reply website/docs/enterprise/entsupport.md]: fixed in recent commit, thanks!

**Review by rissson — APPROVED** [review]
All looks good for the customer portal. I'll let Jens check for the license validation on authentik's side

**Pull request merged**